.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -20,19 +19,18 @@ These points need to be fulfilled for every PR.
   - [ ] reformat the code with `scripts/dev/reformat-all`
   - [ ] make all unit tests pass
   - [ ] fix all memleaks
+  - [ ] fix the CI itself (or rebase if already fixed)
 - [ ] The PR is rebased with current master.
 
 <!--
-If you have any troubles fulfilling these criteria, please write
-about the trouble as comment in the PR. We will help you,
-but we cannot accept PRs that do not fulfill the basics.
+If you have any troubles fulfilling these criteria, please write about the trouble as comment in the PR.
+We will help you, but we cannot accept PRs that do not fulfill the basics.
```

Comment:
This this is a better case for labels.

```suggestion
If you have any troubles fulfilling these criteria, please write about the trouble as comment in the PR.
We will help you, but we cannot accept PRs that do not fulfill the basics.
If you can, please also add the label "build problems" in case we miss your comment.
```

Reply:
We see anyway if the build server fails. Calling for help is better with @mention.

Reply:
Yes, @ mentions are better, but to avoid being flooded with notifications, we definitely should suggest that. But IMO a label would clearly indicate that help is wanted. We could also just create a generic "help wanted" label. Just seeing the failed build server job, doesn't tell you whether the author of the PR can fix it themselves.